src/commands/src/helpers/dependency-helpers.ts

```diff
@@ -63,36 +72,29 @@ export async function downloadBuildDependencies(buildDependencies: BuildDependen
       }
       commands.push(`mkdir -p ${targetDir}`);
       await Promise.all(
```

Comment:
Note: I did not use `allSettled` here because it seemed to not exist with this version of Javascript?

Reply:
What version of node are you using?

Reply:
v18.17.1!